packages/es-components/src/components/containers/notification/useNotification.js

```diff
@@ -43,6 +43,7 @@ function NotificationContent(props) {
     iconColor,
     color,
     dismissNotification,
+    alwaysShowIcon,
```

Comment:
Since the downstream user will be importing `Notification`, `LightNotfication`, or `MessageNotification`, we should add this prop to their respective `.propTypes` and `.defaultProps`, along with a description for the documentation to display.

Reply:
I added those proptypes to the notification types. I also fixed the comments on the other proptype descriptions so they display.